### DISCUSSION TITLE
How do I configure Renovate to raise a single PR for updates across multiple dependency specs?

### DISCUSSION THREAD

**Discussion by allanlewis** [renovatebot/renovate#24160] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

A single PR updating a given set of dependencies (which I can list in the config) across multiple components, i.e. multiple dependency specs.

### What you tried so far.

Nothing, as I'm not sure where to start

### Relevant debug logs

N/A

**Comment by rarkins** [comment]
Use packageRules with matchPackageNames and groupName

**Reply by allanlewis** [reply]
I'm already doing something like this for other purposes and I get one PR per manifest: are you sure that approach will give me one PR updating given dependencies across my monorepo?

**Reply by rarkins** [reply]
Define manifest, we don't use that term in renovate features

**Reply by allanlewis** [reply]
I mean a dependency spec, like a `pyproject.toml` file. I have a bunch of microservices and, while we update dependencies in each separately, there are a few we like to keep consistent. I want to configure Renovate to raise a single PR to update those dependencies across multiple projects in my monorepo. I hope that makes sense 🙏 

And thanks for the quick responses!

**Reply by rarkins** [reply]
By default, Renovate will update across all folders/locations in one PR. If you're not seeing that, it's most likely because you or someone in your company has configured an option like `additionalBranchPrefix` which splits PRs per folder

**Reply by allanlewis** [reply, marked as answer]
That must be it: we're using `additionalBranchPrefix` at the top level. I'll try moving that to package rules so that the default is to put everything in one PR...
